src/lib/learnerrecord.ts

```diff
@@ -72,6 +73,18 @@ export async function bookEvent(
 	}
 }
 
+export async function getCancellationReasons(user: any): Promise<any> {
+	try {
+		return await http.get(`/event/booking/userCancellationReasons`, {
+			headers: {
+				Authorization: `Bearer ${user.accessToken}`,
+			},
+		})
+	} catch (e) {
+		logger.error(e)
+	}
+}
```

Comment:
Can you handle this await using then/catch, instead of a try/catch? If an error occurs getting the cancellation reasons, the app will just hang.